src/test/java/org/minions/devfund/abel/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,167 @@
+package org.minions.devfund.abel.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class for Movie Database Unit Tests.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+
+    /**
+     * SetUp method.
+     */
+    @Before
+    public void setUp() {
+        movieDatabase = new MovieDatabase();
+    }
+
+    /**
+     * Test AddMovie.
+     */
+    @Test
+    public void testAddMovieNewMovieAndActors() {
+        final String expectedMovie = "NewMovie";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "NewMovie" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)